packages/salesforcedx-vscode-apex-replay-debugger/README.md

```diff
@@ -75,36 +72,53 @@ TIP: If your log file is part of your Salesforce DX project, you don’t need to
-1. Step through your code and examine the states of your variables in the VARIABLES section of the Debug view. For details, see [Debugging](https://code.visualstudio.com/docs/editor/debugging) in the Visual Studio Code docs.
+1. Step through your code and examine the states of your variables in the VARIABLES section of the Debug view. For details, see [Debugging](https://code.visualstudio.com/docs/editor/debugging) in the Visual Studio Code docs.  
+As you step through your code during a debugging session, Apex Replay Debugger provides details about your variables from heap dumps on lines where you set checkpoints.  
 1. When you’ve stepped through all the logged events, the debugging session ends. If you want to start again at the beginning of the log, run **SFDX: Launch Apex Replay Debugger with Last Log File**.
 
 ## Considerations
 Keep these considerations and known issues in mind when working with Apex Replay Debugger.
 
-* You can replay only one debug log at a time. This limitation can make it difficult to debug asynchronous Apex, which produces multiple debug logs.
-* You can’t replay a debug log generated by scheduled Apex.
-* Long string variable values are truncated.
-* When viewing a standard or custom object, you can drill down only to its immediate child variables (one level deep).
-* You can’t expand a collection (a list, set, or map), because its members are shown in their string form.
-* Modifying a collection does not update the collection variable in the VARIABLES section of the Debug view.
-* You can’t set method or conditional breakpoints.
-* You can’t evaluate or watch variables or expressions in the Debug view’s WATCH section.
-* While debugging, right-clicking a variable in the VARIABLES section of the Debug view and selecting **Copy Value** works properly. However, **Copy as Expression** and **Add to Watch** don’t work as expected. 
-  * **Copy as Expression** functions like Copy Value: It copies the variable’s value instead of copying the full variable name.
-  * **Add to Watch** copies the variable’s value into the WATCH section, but because we don’t evaluate variables in this section you see only `<VariableValue>:<VariableValue>`.
+- You can use this debugger only in your orgs. ISV customer debugging is unavailable in Apex Replay Debugger. To debug customers’ orgs, use [ISV Customer Debugger](https://marketplace.visualstudio.com/items?itemName=salesforce.salesforcedx-vscode-apex-debugger).
+- You can replay only one debug log at a time. This limitation can make it difficult to debug asynchronous Apex, which produces multiple debug logs.
+- Be sure to start a session soon after uploading your checkpoints, because checkpoints expire after 30 minutes.
+- Be sure to debug your code soon after starting the session, because heap dumps expire about a day after you generate them.
```

Comment:
Everything in this list below the line about heap dump expiration is unchanged. It shows as changed because I changed the asterisks to hyphens.